src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -278,18 +305,20 @@ bamm:PropertyShape
                   ( ?providedType != ?characteristicDataType ) &&
                   not exists { ?providedType bamm:canBeCastTo* ?characteristicDataType }
                )
+               bind( 'ERR_WRONG_DATATYPE' as ?code )
             }
          """
       ] ;
       sh:sparql [
          a sh:SPARQLConstraint ;
-         sh:message "An example value may not be set on a Property which does not have a Characteristic" ;
+         sh:message "An example value may not be set on a Property '{$this}' which does not have a Characteristic" ;
```

Comment:
I don't understand. Please clarify with @atextor , why do we have this error? How is it possible to have a propery without a characteristic? 
Maybe we should write "Property '{$this}' does not have a Characteristic." because this is the actual problem (defining an exampleValue is never the actual problem, am I right?)